tests/fixtures/sdk_code/feature_fraction.py

```diff
@@ -57,7 +57,7 @@
     ),
     skip_fill_na=True,
 )
-feat_1 = grouped_1["sum_item_amt_plus_one_over_30d"]
-feat_2 = feat / feat_1
-feat_2.name = "fraction"
-output = feat_2
+feat_3 = grouped_1["sum_item_amt_plus_one_over_30d"]
+feat_4 = feat_2 / feat_3
+feat_4.name = "fraction"
+output = feat_4
```

Comment:
As we keep the output of item aggregate as feature, the SDK generation will name the variable as `feat` rather than `col`.